packages/components/src/index.js

```diff
@@ -97,7 +97,7 @@ export { default as __experimentalNavigationGroup } from './navigation/group';
 export { default as __experimentalNavigationItem } from './navigation/item';
 export { default as __experimentalNavigationMenu } from './navigation/menu';
 export {
-	Navigator as __experimentalNavigator,
+	NavigatorRoot as __experimentalNavigatorRoot,
```

Comment:
To be honest I don't like the name `NavigatorRoot`. I feel it can be confused with the "Root Screen". In other places in the UI Components we use `Provider` so maybe `NavigatorProvider` would be better for me. That said, don't take my word for it, I know I'm not great at naming things :P

Reply:
Naming things is hard, and doing so in a project like Gutenberg is even harder! `NavigatorProvider` sounds good to me, I'll go ahead and make changes